.gitignore

```diff
@@ -11,3 +11,4 @@ apps/worker/doc/edoc.css
 doc/users-guide/overview-eps-converted-to.pdf
 integration-coverage/
 .rebar
+rebar.lock
```

Comment:
Yeah, I know how it works. It's just that I mostly code libraries instead of "standalone" applications. And for libraries you usually don't check-in the lock-files to avoid versioning conflicts.